lib/core/src/lib/app-config/provide-app-config.ts

```diff
@@ -0,0 +1,43 @@
+/*!
+ * @license
+ * Copyright © 2005-2025 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { inject, provideAppInitializer } from '@angular/core';
+import { StoragePrefixFactory } from './app-config-storage-prefix.factory';
+import { loadAppConfig } from './app-config.loader';
+import { AppConfigService } from './app-config.service';
+import { StorageService } from '../common';
+import { AdfHttpClient } from '@alfresco/adf-core/api';
```

Comment:
Will be that path working? You are in core lib and if I'm not wrong earlier we had a problem with imports from the same lib

Reply:
This one is questionable, as API is in the secondary package entry

<img width="354" alt="Screenshot 2025-07-08 at 7 43 05 AM" src="https://github.com/user-attachments/assets/d20ac3fb-7b0f-41d7-afd5-d2dccae82f44" />